index.html

```diff
@@ -1472,6 +1472,193 @@ <h2>
         feature/default allowlist=] is [=default allowlist/'self'=].
       </p>
     </section>
+    <section data-cite="webdriver WebDriver-BiDi">
+      <h2>
+        Automation
+      </h2>
+      <p>
+        For the purposes of user-agent automation and application testing, this
+        document defines extensions to the [[[WebDriver]]] and
+        [[[WebDriver-BiDi]]] specifications. It is OPTIONAL for a user agent to
+        support them.
+      </p>
+      <pre class="idl">
+        dictionary GeolocationMockCoordinates {
+          double accuracy = 1;
+          required double latitude;
+          required double longitude;
+          double? altitude = null;
+          double? altitudeAccuracy = null;
+          double? heading = null;
+          double? speed = null;
+        };
+      </pre>
+      <p>
+        To <dfn>set the system coordinates</dfn> given an optional
+        {{GeolocationMockCoordinates}}
+        |coordinates:GeolocationMockCoordinates|, and an optional [=user
+        agent=] |agent|:
+      </p>
+      <ol class="algorithm">
+        <li>Let |targets| be a list containing all [=environment settings
+        object=] which belong to the |agent| if provided, or all [=user
+        agents=] otherwise.
+        </li>
+        <li>Let |tasks| be an [=list/empty=] [=list=].
+        </li>
+        <li>For each |target| in |targets|:
+          <ol>
+            <li>Queue a task |task| on the [=geolocation task source=] of
+            |target|'s [=relevant settings object=]'s [=global object=]'s
+            [=Window/browsing context=] to:
+              <ol>
+                <li>If |coordinates| is not given, clear the system location
```

Comment:
We should define "system location" in a way that ties into the "acquire a position" algorithm.

Reply:
I thought about this as I was specifying this, but I think we might want keep them separate. 

I initially tried to hook into "acquire a position"... but I think it might be best to keep them separate: this algorithm just sets the coordinates for the entire user agent*, then the actual API methods "acquire a position". 

*at least initially... let's start simple and then see if we need to set it per origin. That adds a lot of complexity thought, so we probably don't want to do that.

Reply:
I think it's as easy as saying, "a top-level traversable has a mock position which is initially unset". This algorithm sets the mock position and the "acquire a position" algorithm checks if the mock position is set in step 5.3.3.

Reply:
Ok, yeah, I like that the mock position to take precedence over the system one... and it doesn't imply that it's touching the system one at all (it's just something that works with automation and that's it).